docs/installation/basic_installation.md

```diff
@@ -58,7 +59,37 @@ If you would prefer less automation, or you  already have existing services on y
 
     <p>Once the preparations are done, the installer will prompt you to run <code>seat install:production</code>. </p>
 
-    <p>To configure ESI for SSO and API pulls, please refer to the <a href="/configuration/esi_configuration/">Configuring ESI</a> documentation page</p>
+</div>
+</section>
+
+!!! success "Installation complete"
+
+    You now have SeAT installed.
+    
+## Next steps
+
+1. [Configure ESI](/configuration/esi_configuration/) for SSO and API pulls
+2. Create a SeAT account with a character of yours
+3. log off or use new browser
+4. create admin link
+
+<section class="mdc-tabs">
+<ul class="mdc-tab-bar">
+  <li class="mdc-tab active"><a role="tab" data-toggle="tab">Docker</a></li>
+  <li class="mdc-tab"><a role="tab" data-toggle="tab">SeAT Tool</a></li>
+</ul>
+<div class="mdc-panels">
+<div role="tabpanel" class="mdc-panel active">
+
+    <pre><code class="bash hljs">docker-compose exec seat-app php artisan seat:admin:login</code></pre>  
+
+</div>
+<div role="tabpanel" class="mdc-panel">
+    
+    <pre><code class="bash hljs">php artisan seat:admin:login</code></pre>
 
 </div>
 </section>
+
+5. assign `Superadmin` to your character/user you created in step 2
+6. Log off with admin (and never use it again)
```

Comment:
* (You won't be needing to login again with this account in the future, unless in an emergency)